src/api/app/controllers/webui/projects/bs_requests_controller.rb

```diff
@@ -12,7 +12,11 @@ def index
         session[:request_numbers] = requests_query.requests.map(&:number)
 
         respond_to do |format|
-          format.json
+          if switch_to_webui2?
+            format.json { render 'webui2/shared/bs_requests/index' }
```

Comment:
this view already exists, right?

Reply:
Right. If you ask why I don't use `switch_to_webui2` here, it's because it didn't work for respond_to blocks.